sec4-notebook.tex

```diff
@@ -1,8 +1,57 @@
 \section{The Notebook Aspect}\label{notebook-aspect}
 
+The Science Platform offers in-house scientists and engineers as well as (in operations) general science users the opportunity to interactively explore the LSST dataset from a python notebook environment.
+
 \subsection{JupyterHub / JupyterLab service}\label{jupyterhub-jupyterlab-service}
 
+The notebook service is a scalable kubernetes deployment with JupyterHub spawning individual JupyterLab individual pods for each user. The basic architecture is outlined in \figref{fig:jlarch}. JupyterLab is a rich notebook architecture that offers terminal access and dashboarding in addition to the traditional Jupyter Notebook concept, and
```

Comment:
It's not clear that kubernetes is capitalised (their web pages use both) and as it's a common noun in Greek it looks wrong to me to do so... Fixed rest.